h2o-py/tests/testdir_algos/stackedensemble/pyunit_stackedensemble_binary.py

```diff
@@ -0,0 +1,115 @@
+        return pu.ns(x=train.columns, y=target, train=train, test=test)
+
+    def train_base_models(self, dataset):
+        nfolds = 3
+        gbm = H2OGradientBoostingEstimator(distribution="bernoulli",
+                                           ntrees=10,
+                                           max_depth=3,
+                                           min_rows=2,
+                                           learn_rate=0.2,
+                                           nfolds=nfolds,
+                                           fold_assignment="Modulo",
+                                           keep_cross_validation_predictions=True,
+                                           seed=seed)
+        gbm.train(x=dataset.x, y=dataset.y, training_frame=dataset.train)
+
+        rf = H2ORandomForestEstimator(ntrees=20,
+                                      nfolds=nfolds,
+                                      fold_assignment="Modulo",
+                                      keep_cross_validation_predictions=True,
+                                      seed=seed)
+        rf.train(x=dataset.x, y=dataset.y, training_frame=dataset.train)
+        return [gbm, rf]
+
+    def train_stacked_ensemble(self, dataset, base_models):
+        se = H2OStackedEnsembleEstimator(base_models=[m.model_id for m in base_models], seed=seed)
+        se.train(x=dataset.x, y=dataset.y, training_frame=dataset.train)
+        return se
+
+
+class StackedEnsembleBlendingTest(StackedEnsembleTest):
+
+    def prepare_data(self):
+        ds = super(self.__class__, self).prepare_data()
+        train, blend = ds.train.split_frame(ratios=[.7], seed=seed)
+        return ds.extend(train=train, blend=blend)
+
+    def train_base_models(self, dataset):
+        gbm = H2OGradientBoostingEstimator(distribution="bernoulli",
+                                           ntrees=10,
+                                           seed=seed)
+        gbm.train(x=dataset.x, y=dataset.y, training_frame=dataset.train)
+
+        rf = H2ORandomForestEstimator(ntrees=10,
+                                      seed=seed)
+        rf.train(x=dataset.x, y=dataset.y, training_frame=dataset.train)
+        return [gbm, rf]
+
+    def train_stacked_ensemble(self, dataset, base_models):
+        se = H2OStackedEnsembleEstimator(base_models=[m.model_id for m in base_models], seed=seed)
+        se.train(x=dataset.x, y=dataset.y, training_frame=dataset.train, blending_frame=dataset.blend)
+        return se
+
+
+def test_suite_stackedensemble_binary_model(blending=False):
+    t = StackedEnsembleTest() if not blending else StackedEnsembleBlendingTest()
+
+    def test_saved_binary_model_produces_same_predictions_as_original():
+        ds = t.prepare_data()
+        base_models = t.train_base_models(ds)
+        se_model = t.train_stacked_ensemble(ds, base_models)
+        
+        #Predict in ensemble in Py client
+        preds_py = se_model.predict(ds.test)
+        
```

Comment:
`pyunit_NOPASS_stackedensemble_binary.py` was failing due to usage of save model from old H2O-3 version.
Fixed it by saving binary model in tmp dir first.